src/components/LocationSearch/LocationSearch.js

```diff
@@ -19,15 +20,13 @@ export default function LocationSearch({ updateQuery, setSearch, city }) {
         return res.json();
       })
       .then((data) => {
-        setLocations(data);
+        setCityList(data);
         setError(null);
```

Comment:
Consider moving this useEffect to AgencyContext (fetch it once when the app loads)

Reply:
👍